distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -112,9 +112,29 @@ public void testNitfImageGeneration() throws Exception {
     assertGetJpeg(REST_PATH.getUrl() + id + "?transform=resource&qualifier=overview");
   }
 
+  @Test
+  public void testNitfImageGenerationThumbnailOnly() throws Exception {
+    configureNitfRenderPlugin(120, false, false);
+    String id = ingestNitfFile(TEST_IMAGE_NITF);
+
+    assertGetJpeg(REST_PATH.getUrl() + id + "?transform=thumbnail");
+    given()
+        .get(REST_PATH.getUrl() + id + "?transform=resource&qualifier=overview")
+        .then()
+        .assertThat()
+        .statusCode(HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
Seems weird this wouldn't be a 404 or something like that.

Reply:
Agree. Maybe another ticket?